.jshintignore

```diff
@@ -8,6 +8,7 @@ apps/calendar/js/ext/**
 apps/camera/js/vendor/**
 apps/camera/test/**
 apps/privacy-panel/js/vendor/**
+apps/**/*.esp.js
```

Comment:
No: `*.esp` is the template file you put in your test directory of the app, which instructs what code (ex: `performance.mark`) should be woven into the code in `build_stage` and the profile. See the demo I put in this patch: ` apps/system/lockscreen/raptor/lockscreen.esp.js`.